pkg/resource/function/hooks.go

```diff
@@ -591,16 +603,11 @@ func (rm *resourceManager) deleteFunctionCodeSigningConfig(
 	return nil
 }
 
-// setResourceAdditionalFields will describe the fields that are not return by
-// GetFunctionConcurrency calls
-func (rm *resourceManager) setResourceAdditionalFields(
+// getFunctionConcurrency will describe the fields that are not return by GetFunctionConcurrency calls
```

Comment:
Is this comment accurate? (First, the function name is wrong (get vs set), but also) are these the fields that are **not** returned by GetFunctionConcurrency?  It looks to me that it's `ReservedConcurrentExecutions`, which is actually returned from GetFunctionConcurrency. (I guess it's a comment that was here before and you didn't change it with this PR, but we can update it)

Reply:
Corrected the name and changed the description to more inline with the function property.